### DISCUSSION TITLE
[Feature] Selective Date Sync, Rolling Window

### DISCUSSION THREAD

**Discussion by StackShard** [immich-app/immich#11271] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Have a setting to automatically backup/sync photos that are older than X days only (leaving new things alone).

Use case, the latest photos have the highest chance to be edited/changed on the device. In an effort to stop duplicates, etc. consider a toggle to only transfer from a rolling date. It won't be perfect to avoid duplicates when photos are edited, it will help greatly. It will also give a solid indication of photos to purge from the device. In this case, it's pretty safe to say that photos "older" than X date can be deleted on device because it's transferred.

Bonus, "delete on transfer" like Photosync would be ideal as well.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4169
